tests/syntax_tests/data/parsing/grammar/expressions/expected/apply.res.txt

```diff
@@ -6,4 +6,7 @@
 ;;List.reduce (fun [arity:2]acc -> fun curr -> acc + curr) 0 myList
 let unitUncurried = apply ()
 ;;call ~a:(a : int)
+;;call (~ a)
+;;call ~a:(a ^ (~ a))
+;;call ~a:(~ a)
 ;;call_partial 3 ...
```

Comment:
Can you add some simple examples too? E.g. `let z = ~y`. To illustrate where parens are added, and where they are not.

Reply:
I think it has been added in the `unary.res` file already?